frodo.js

```diff
@@ -1,40 +1,36 @@
-/**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?